--- unifi-video.orig	2018-09-24 08:26:15.494409541 -0700
+++ unifi-video	2018-09-24 08:26:47.327743681 -0700
@@ -263,7 +263,7 @@
 		 -Dcom.sun.management.jmxremote.authenticate=false \
 		 -Dcom.sun.management.jmxremote.port=${JVM_JMXREMOTE_PORT}"
 	[ -z "${JVM_JMXREMOTE_HOST}" ] && \
-		JVM_JMXREMOTE_HOST=$(hostname -I | cut -d' ' -f1)
+		JVM_JMXREMOTE_HOST=$(hostname -i | cut -d' ' -f1)
 	[ -z "${JVM_JMXREMOTE_HOST}" ] || \
 		JVM_OPTS="${JVM_OPTS} -Djava.rmi.server.hostname=${JVM_JMXREMOTE_HOST}"
 
@@ -319,7 +319,6 @@
 	start)
 	require_root
     update_limits
-        ulimit -H -c 200
         echo 0x10 > /proc/self/coredump_filter
 		if is_service_running "${PIDFILE}" >/dev/null; then
 			log_verbose "${NAME} is already running..."
@@ -335,7 +334,6 @@
 		;;
 	stop)
  	require_root
-        ulimit -H -c 200
         
         log_verbose "Backing up ${DATADIR}/system.properties in ${BASEDIR}/etc/system.properties"
         cp -f "${DATADIR}/system.properties" "${BASEDIR}/etc/system.properties"
